Documentation/gettingstarted/kube-router.rst

```diff
@@ -82,7 +82,9 @@ Deploy Cilium
 In order for routing to be delegated to kube-router, tunneling/encapsulation
 must be disabled. This is done by setting the ``tunnel=disabled`` in the
 ConfigMap ``cilium-config`` or by adjusting the DaemonSet to run the
-``cilium-agent`` with the argument ``--tunnel=disabled``:
+``cilium-agent`` with the argument ``--tunnel=disabled``. Moreover, in the
+same ConfigMap, we must explicitely set ``ipam: kubernetes`` since kube-router
```

Comment:
Typo: “explicitely” (I always do that one, too >_<). Can you please fix it?

```suggestion
same ConfigMap, we must explicitly set ``ipam: kubernetes`` since kube-router
```

Reply:
doh! good catch